### PR TITLE
Configure API endpoint

### LINKED ISSUE #178: Make API address of Dashboard configurable
Currently, the dashboard expects the engine on the current path on `/v1`, but if you are running Docker, you are running into problems because a different "host" or at least port is used.

### PR DESCRIPTION
**Pull request by lebogg**
Resolves #178

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { sveltekit } from '@sveltejs/kit/vite';
+import type { UserConfig } from 'vite';
+
+const config: UserConfig = {
+	plugins: [sveltekit()],
+
+	server: {
+		// proxy: {
```
Comment (oxisto) [inline vite.config.ts:8]: You need to re-enable this for easier development to work again for npm run.

File: .env.development [hunk 2]
```diff
@@ -1,5 +1,6 @@
-VITE_AUTH_URL=http://localhost:8080/v1/auth/authorize
-VITE_TOKEN_URL=http://localhost:8080/v1/auth/token
-VITE_CLIENT_ID=dashboard
-VITE_REDIRECT_URI=http://localhost:5173/callback
-VITE_SCOPE=profile email
+PUBLIC_AUTH_URL=http://localhost:8080/v1/auth/authorize
```
Comment (oxisto) [inline .env.development:1]: can we prefix them with `OAUTH` when we are changing them anyway?

**Review by oxisto — COMMENTED** [review]
Just smaller comments

**Comment by oxisto** [comment]
Reopened to trigger CI

File: .env.development (hunk 2 above)
Comment (lebogg) [reply .env.development]: Like now?

Comment (oxisto) [reply .env.development]: I thought you need the PUBLIC prefix, otherwise it won't work? I meant like `PUBLIC_OAUTH....`

**Comment by lebogg** [comment]
Can we merge it?

**Review by oxisto — APPROVED** [review]

**Pull request merged**